src/patient/PatientDoctors.js

```diff
@@ -114,6 +116,7 @@ const PatientDoctors = () => {
           </TableBody>
         </Table>
       </TableContainer>
+      <InfoPopover style={{ marginTop: '15px', marginBottom: '5px' }}>{addDoctorInfo}</InfoPopover>
       <Button onClick={handleOpen} disableElevation disableRipple disableFocusRipple style={btnStyle} sx={{ flexDirection: 'column' }}>
```

Comment:
Q: Is this part of the tutorial?
FIX: Place above table

Reply:
Moved it, thanks! Is what part of the tutorial? There wasn't much of a tutorial, so likely no